src/lib/registry/logger-registry.js

```diff
@@ -0,0 +1,58 @@
+const Logger = require('../logger');
+const EmptyRegistry = require('./empty-registry');
+const { throwIf } = require('../utils/type-checks');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
const { throwIf } = require("../utils/type-checks");
```